app/models/miq_report/formatting.rb

```diff
@@ -97,7 +97,7 @@ def format(col, value, options = {})
     options[:column] = col
 
     # Chargeback Reports: Add the selected currency in the assigned rate to options
-    if db.to_s == "ChargebackVm" || db.to_s == "ChargebackContainerProject"
+    if Chargeback.db_is_chargeback?(db)
```

Comment:
Is this supposed to be true for `Chargeback` db value too?

```ruby
irb(main):002:0> Chargeback.db_is_chargeback?("Chargeback")
=> false
irb(main):003:0> Chargeback.db_is_chargeback?("ChargebackVm")
=> true
```

Note, this autoloads Chargeback and it's subclasses to call that method.  I guess that's ok, but we do this even if you never use chargeback reports.